peps/pep-0771.rst

```diff
@@ -70,6 +89,32 @@ or backend package. There is currently no mechanism in Python packaging
 infrastructure to disallow conflicting or incompatible extras to be installed,
 and this PEP does not change that.
 
+Real-world examples of packages that allow users to select different backends or
+frontends with extras include:
```

Comment:
Likewise, I wonder if there's a better (more concise) way of integrating the description of the pattern with the examples.

Reply:
Done